services/site/src/pages/about.tsx

```diff
@@ -0,0 +1,193 @@
+            writing meaningful markup and completing quizzes.
+          </p>
+
+          <h2 className="h4 mt-8">Coding-Challenges and Quizzes</h2>
+
+          <p>
+            Various accessibility topics are split up into short challenges. On <span aria-label="Allyphant">a11yphant</span>, you solve coding levels
+            and single-choice quizzes in each challenge. Each level teaches you one thing at a time. Our evaluation provides you with detailed
+            feedback on our success criteria. If you struggle to solve a challenge, hints will help you out.
+          </p>
+
+          <h2 className="h4 mt-8">Meet the team</h2>
+
+          <p>
+            <span aria-label="Allyphant">a11yphant</span> started as a master’s project created by six students at the{" "}
+            <a href="https://www.fh-salzburg.ac.at/en/" target="_blank" rel="noopener noreferrer nofollow">
+              Salzburg University of Applied Sciences (Austria)<span className="sr-only">(opens in a new tab)</span>
+            </a>
+            .<span className="font-medium">Concept and Development</span> by <a href="https://dnikub.dev">Daniela Kubesch</a>,{" "}
+            <a href="https://lucapircher.at/" target="_blank" rel="noopener noreferrer nofollow">
+              Luca Pircher<span className="sr-only">(opens in a new tab)</span>
+            </a>
+            ,{" "}
+            <a href="https://github.com/thomasdax98" target="_blank" rel="noopener noreferrer nofollow">
+              Thomas Dax<span className="sr-only">(opens in a new tab)</span>
+            </a>{" "}
+            and{" "}
+            <a href="https://github.com/hntrhfr" target="_blank" rel="noopener noreferrer nofollow">
+              Michael Hinterhofer<span className="sr-only">(opens in a new tab)</span>
+            </a>
+            .<span className="font-medium">Interface and Corporate Design</span> by{" "}
+            <a href="https://johannawicht.com/" target="_blank" rel="noopener noreferrer nofollow">
+              Johanna Wicht<span className="sr-only">(opens in a new tab)</span>
+            </a>{" "}
+            and{" "}
+            <a href="https://www.fabianhellerdesign.com/" target="_blank" rel="noopener noreferrer nofollow">
+              Fabian Heller<span className="sr-only">(opens in a new tab)</span>
+            </a>
+            .
+          </p>
+
+          <p>
+            As we hope to make the world more inclusive, we continue developing <span aria-label="Allyphant">a11yphant</span> as an Open Source
+            project. So, if you find an accessibility issue or want to contribute or support us in any other way, please{" "}
+            <a href="mailto:info@a11yphant.com">let us know</a>.
+          </p>
+
+          <h2 className="mt-14 h3">What others say about us:</h2>
+
+          <ul className="mt-4">
```

Comment:
Do we maybe want to add a list-icon/indicator? I think here it could actually make sense.